remote/remote_test.go

```diff
@@ -720,17 +842,17 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 			var tarPath string
 
 			it.Before(func() {
-				baseImage, err := remote.NewImage(repoName, authn.DefaultKeychain)
+				existingImage, err := remote.NewImage(repoName, ggcrauthn.DefaultKeychain)
 				h.AssertNil(t, err)
 
-				h.AssertNil(t, baseImage.SetLabel("mykey", "oldValue"))
-				h.AssertNil(t, baseImage.Save())
+				h.AssertNil(t, existingImage.SetLabel("mykey", "oldValue"))
+				h.AssertNil(t, existingImage.Save())
 
 				tarFile, err := ioutil.TempFile("", "add-layer-test")
 				h.AssertNil(t, err)
 				defer tarFile.Close()
 
-				tarPath, err = h.CreateSingleFileTar("/new-layer.txt", "new-layer")
+				tarPath, err = h.CreateSingleFileLayerTar("/new-layer.txt", "new-layer")
 				h.AssertNil(t, err)
```

Comment:
We don't appear to be using any of this setup logic in the tests.

Reply:
Good catch! I moved the only bit that was used the test that used it.